.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Release Checklist
+about: All the tasks required to complete a release across languages
+title: Release v<fill in>
+labels: ''
+assignees: ''
+
+---
+
+Full release instructions are at: [RELEASE.md](https://github.com/sigstore/protobuf-specs/RELEASE.md)
+
+## Pre Release
+- [ ] Check mediatype version of [Bundle](https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_bundle.proto)
+- [ ] Check mediatype version of [TrustedRoot](https://github.com/sigstore/protobuf-specs/blob/main/protos/sigstore_trustroot.proto),
+- [ ] Update [pyproject.toml](gen/pb-python/pyproject.toml) so the `version` matches the targeted release.
+
```

Comment:
Can we add a checklist for things to _do_ for the release process?

- (release manager) `git tag v<fill in>`, `release/java/v<fill in>`, `release/python/v<fill in>`

I think the only person that needs to do anything here is the Java person (aka you/Bob)

Then post release there should be a quick way to verify: 
- https://pypi.org/project/sigstore-protobuf-specs/
- something for Java